src/models/createNewHouseModel.js

```diff
@@ -0,0 +1,103 @@
+const mongoose = require('mongoose')
+
+const housesSchema = new mongoose.Schema({
```

Comment:
`houseSchema` will also need a status field which would store the information if a house was deleted, sold or is still active. You could assign the numbers 0, 1, 2 to the corresponding statuses and store them in the database.